sdk/storage/CONTRIBUTING.md

```diff
@@ -10,9 +10,9 @@ The Azure Storage development team uses Visual Studio Code. However, any preferr
 
 ### Install
 
-* Node.js valid LTS versions (>=6.5.0)
-* Browsers like Chrome, Edge or Firefox
-* Clone the source code from GitHub
+- Node.js valid LTS versions (>=6.5.0)
```

Comment:
LTS version now >=8.16.0

Reply:
we probably need a work item to update node LTS version for the whole repo.

Reply:
#4088